content/en/network_performance_monitoring/devices/guide/build-ndm-profile.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Build a NDM Profile
+kind: guide
+further_reading:
+- link: "https://datadoghq.dev/integrations-core/tutorials/snmp/profile-format/"
+  tag: "Documentation"
+  text: "Profile format reference"
+- link: "https://datadoghq.dev/integrations-core/tutorials/snmp/sim-format/"
+  tag: "Documentation"
+  text: "Simulation data format reference"
+---
+
+Datadog Network Device Monitoring uses profiles for certain makes and models of network devices. This tutorial shows the steps for building a basic NDM profile that collects OID metrics from HP iLO4 devices.
+
+NDM profiles use SNMP concepts. For basic details on SNMP, refer to the [terminology][1].
+
+<div class="alert alert-warning">
+This guide is for advanced users. Most devices can be configured using the <a href="/network_performance_monitoring/devices/profiles#metric-definition-by-profile">Datadog profiles</a>.
+</div>
+
+## Research
+
+The first step to building a NDM profile is researching the device and determining the metrics to collect.
+
+### Device information
+
+Refer to the manufacturer's website or search the web to find the following information:
+
+- Device name, manufacturer, and [system object identifier][1].
```

Comment:
I think we need to mention MIB (ASN1, textual format) too here. Even though the integration doesn't require the MIB, it's essential for understanding the available metrics and how they are structured (Tables, Indexes, OIDs).

Reply:
that's a great note, we should also mention that they do not need to ship the MIB with the profile in order to collect metrics from it

Reply:
It's mentioned in the list (4th bullet point) but I will update with additional info.